detox/test/ios/example/AppDelegate.m

```diff
@@ -136,10 +136,21 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 #else
 	jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
 #endif
-	
+
+  // Parse process arguments into a launchArgs initialProp that is passed to the react root view
+  NSMutableArray *arguments   = [[NSMutableArray alloc]initWithArray:[[NSProcessInfo processInfo] arguments]];
+  [arguments removeObjectAtIndex:0]; // Remove executable path
+  NSMutableDictionary *initialProps = [NSMutableDictionary dictionary];
+  if([arguments count] > 0) {
+    NSString *allArgs = [arguments componentsJoinedByString:@" "];
+    initialProps[@"launchArgs"] = allArgs;
+  } else {
+    initialProps = NULL;
+  }
+
 	RCTRootView *rootView = [[RCTRootView alloc] initWithBundleURL:jsCodeLocation
 														moduleName:@"example"
-												 initialProperties:nil
+												 initialProperties:initialProps
```

Comment:
I am sorry, I don't understand this change. Launch arguments are not initial props. Why was this code added here? 

Launch arguments are process launch argument.